app/views/appliances/destroy.js.erb

```diff
@@ -0,0 +1,3 @@
+$("#<%= dom_id(@appliance) %>").fadeOut(function() {
+  $(this).remove();
+});
```

Comment:
I think this isn't removing everything it needs to. The id of the appliance is on the card, but it's _not_ on the div column— meaning if I delete an appliance in the middle of a row, the empty space remains.

![remove-appliance](https://user-images.githubusercontent.com/17581658/171906207-2222a229-bb5b-46a2-abde-8230e6776d28.gif)

So you could put an id on the div column instead or use a different selector, but this can still run into issues if appliances in two different rows are deleted: 

<img width="1125" alt="Screen Shot 2022-06-03 at 11 26 06 AM" src="https://user-images.githubusercontent.com/17581658/171906461-b9810457-c8b6-4bb6-916d-bfcab5ecffde.png">

So I think actually the better solution is to use one div row for **every** appliance, including the form. Bootstrap column classes are smart enough so that if more than 3 `col-md-4`s appear in one row, the fourth column will appear on the next line. More on that in the final review comment.